shell.nix

```diff
@@ -40,7 +40,6 @@ let
     ];
 
     haskell = [
-      unstable.haskell-language-server
```

Comment:
`LSP :: The following servers support current file but do not have automatic installation configuration: lsp-haskell` 

but was it working for you?

Reply:
Weird. Yeah, I have `lsp-haskell` via the `doom-emacs` module.

Reply:
From the `lsp-haskell` [docs](https://github.com/emacs-lsp/lsp-haskell#emacs-configuration):

> lsp-mode automatically requires the lsp-haskell package , so you do not need to require lsp-haskell unless you like being explicit.

Reply:
🤦‍♀️ ZOMG no, I have it in my `home-manager`

Reply:
Also, landing soon:

> haskell-language-server: 0.9.0 -> 1.0.0 

https://github.com/NixOS/nixpkgs/commit/3c81fd41eec6e285c1e0e3a17fa2df39d74e911c